client/history/metricClient.go

```diff
@@ -454,6 +454,23 @@ func (c *metricClient) ReplicateRawEvents(
 	return err
 }
 
+func (c *metricClient) ReplicateEventsV2(
+	context context.Context,
+	request *h.ReplicateEventsV2Request,
+	opts ...yarpc.CallOption) error {
```

Comment:
nit:

```
	opts ...yarpc.CallOption,
) error {
```

Reply:
going to keep this as is, since manual formatting is slow according to exp
we should actually consider using existing tool + rule to format all src